src/test/ui/error-codes/E0775.rs

```diff
@@ -0,0 +1,11 @@
+// only-armv4t-unknown-linux-gnueabi
+// only-armv5te-unknown-linux-gnueabi
+// only-armv5te-unknown-linux-musleabi
```

Comment:
Are you sure you can stack `only` like that?
I recall it did not work and tests like that have been changed to use different conditions.

Reply:
I have no idea, I couldn't really find any documentation on how the comments worked and figured it was worth trying before writing the mass of `ignore-*` I'd need to do to run using just those 3 architectures

Reply:
Judging by the comment it won't run this test on any target: https://github.com/rust-lang/rust/blob/6cae28165f0450fd3f100374b26841e458b8cfef/src/tools/compiletest/src/header.rs#L53